### PR TITLE
dom-event-delegation

### PR DESCRIPTION
**Pull request by jghil**
dom-event-delegation

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var $taskList = document.querySelector('.task-list');
+
+$taskList.addEventListener('click', function (event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+  if (event.target && event.target.nodeName === 'BUTTON') {
+    var $closestElement = event.target.closest('.task-list-item');
+    console.log($closestElement);
+    $closestElement.remove($closestElement);
```
Comment (BrettAlbright) [inline dom-event-delegation/main.js]: the remove method does not take an argument, it is just called on the element you want to remove.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

File: dom-event-delegation/main.js (hunk 1 above)
Comment (jghil) [reply dom-event-delegation/main.js]: gotcha! i removed it! thanks Brett

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**